clayui.com/src/styles/_404-page.scss

```diff
@@ -0,0 +1,66 @@
+.clay-404-page-container {
+	background-image: url('../../static/images/home/bg_white.svg');
+
+	header {
+		background-color: #fff;
+
+		img {
+			width: 32px;
+		}
+	}
+
+	.navbar {
+		width: 100%;
+	}
```

Comment:
Remove this

Reply:
Having this `.navbar { width: 100%; }` goes hand in hand with the lack of wrapper you proposed. See my comment [above](https://github.com/liferay/clay/pull/3201/commits/ce0bddf0daf359c4ab2849bfdb68eec30f884b4b#r422001714)